src/app/admin/adminApp.js

```diff
@@ -37,6 +38,7 @@ export default function AdminPage() {
         edit={EditGuesser}
         show={ShowGuesser}
       />
+      <Resource name="Faq" options={{ label: 'FAQ' }} list={ListFaq} show={ShowGuesser} edit={EditGuesser} />
       <Resource
         name={RESOURCES.feedback}
         options={{ label: 'Feedback' }}
```

Comment:
Change the resource name to match the other resources. Add faq property in the RESOURCES and use it here

Reply:
Thanks for comment! Fixed this.